tensorflow/python/training/experimental/mixed_precision.py

```diff
@@ -73,39 +73,116 @@ def _wrap_optimizer(opt, loss_scale):
 
 @tf_export(v1=['train.experimental.enable_mixed_precision_graph_rewrite'])
 def enable_mixed_precision_graph_rewrite(opt, loss_scale='dynamic'):
-  """Enable mixed precision via a graph rewrite.
-
-  Mixed precision is the use of both float16 and float32 when training a model,
-  and is used to make the model run faster. This function will use mixed
-  precision to speed up the execution time of your model when run on a GPU. It
-  does this by changing the dtype of certain operations in the graph from
-  float32 to float16.
-
-  This function additionally wraps an Optimizer with a LossScaleOptimizer, which
-  is required to prevent underflow in the float16 tensors during the backwards
-  pass. An optimizer must be passed to this function, which will then be wrapped
-  to use loss scaling.
+  """Enables faster training on compatible hardware accelerators by using a
+  mix of numerical data types. Known as mixed precision, this is achieved
```

Comment:
Please make sure the brief description is only a single line.

Reply:
Just to clarify, the brief description has to be within 80 characters?

Reply:
Actually a bit less because of the indent and """. But yes, the first line of the docstring is brief, then a blank line, then the description.

Reply:
I have made the requested changes!